Automattic/woocommerce-payments
# Update `subscriptions-core` to 6.4.0

This PR bumps the `subscriptions-core` version to 6.4.0 and adds changelog entries for any changes that are relevant to WooPayments.

```
* Add - Use admin theme color and the correct WooCommerce colors across subscription interfaces.
* Fix - When HPOS is enabled, fix quick-editing the subscription statuses on the admin list table.
* Fix - Ensure subscriptions have a date created that correctly accounts for the site's timezone. Fixes issues with subscriptions having a date created double the site's UTC offset.
* Fix - Resolve an issue that would cause 3rd party plugin edit product fields with the show_if_variable-subscription class to be incorrectly hidden.
* Fix - Prevent subscription from turning into manual renewal after deleting payment method.
* Dev - Update subscriptions-core to 6.4.0.
```

### Testing instructions

1. Checkout this branch
2. Run composer install
3. Confirm `vendor/woocommerce/subscriptions-core` is running the latest
